src/extension/extension.ts

```diff
@@ -297,7 +297,7 @@ export async function activate(context: vs.ExtensionContext, isRestart: boolean
 
 		flutterDaemon = new FlutterDaemon(logger, workspaceContext as FlutterWorkspaceContext, flutterCapabilities, runIfNoDevices, portFromLocalExtension);
 
-		deviceManager = new FlutterDeviceManager(logger, flutterDaemon, config, workspaceContext, runIfNoDevices);
+		deviceManager = new FlutterDeviceManager(logger, flutterDaemon, config, workspaceContext, runIfNoDevices, portFromLocalExtension);
```

Comment:
It might be a little clearer to name this something like `daemonPortOverride` which describes what it is rather than just where it's from.

Reply:
I renamed this in the device manager instead

Reply:
Those changes look good, thanks!